bpf/process/generic_calls.h

```diff
@@ -27,6 +27,11 @@ generic_process_event(void *ctx, int index, struct bpf_map_def *heap_map,
 	if (!config)
 		return 0;
 
+	index = e->index;
```

Comment:
Do we still need the assembly here? Something like `index = e->index & 7` would not work?

Reply:
yep, I tried that but 5.4 fails with that.. extra assembly instead seems to work